docs/userGuide/cliCommands.md

```diff
@@ -122,3 +123,17 @@ MarkBind Command Line Interface (CLI) can be run in the following ways:
 * `markbind serve`
 * `markbind serve ./myWebsite`
 * `markbind serve -p 8888 -s otherSite.json`
+
+### `--help` Usage
+
+**Format:** `markbind [<command>] --help`<br>
+**Alias:** `markbind [<command>] -h`
+
+**Description:** Provides general help information for `markbind` or specific help information for one of its commands if a command (or its alias) precedes the help argument.
```

Comment:
I think help information sounds a little weird. How about "provides detailed usage guide for the given command preceding the help argument"